src/sandstorm/run-bundle.c++

```diff
@@ -290,7 +292,17 @@ public:
     }
   }
 
-  ~CurlRequest() {
+  explicit CurlRequest(kj::StringPtr url, int outFd): url(kj::heapString(url)) {
+    KJ_SYSCALL(pid = fork());
+    if (pid == 0) {
+      KJ_SYSCALL(dup2(outFd, STDOUT_FILENO));
+      KJ_SYSCALL(execlp("curl", "curl", isatty(STDERR_FILENO) ? "-f" : "-fs",
```

Comment:
We also rely on `tar`, `xz`, and `id` from the host. The thing is, when we're executing outside our chroot, we can't easily run bundled binaries, because they might depend on a different libc, etc. The updater has to run outside the chroot because it needs to download a new chroot.